portfolio/src/main/webapp/index.html

```diff
@@ -1,47 +1,53 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <header>
-      <h1 id="title" class="center">Juan Uribe</h1>
-      <a href="contact.html"><p id="contact">Contact Juan</p></a>
-    </header>
-    <div id="content">
-      <p>This is my portfolio</p>
-      <img src="/images/profileImage.JPG" alt="Profile image of Juan Uribe" id="profile-image">
-      <p>But before you continue, here is a little introduction about myself.</p>
-      <p>I was born in Bogota, Colombia, where I grew up and went to school. It wasn't until 2019 that I moved to LA, to start my college life at Loyola Marymount University.</p>
-      <p>Click here to get a random fact about myself:</p>
-      <button onclick="addRandomFact()">Fact Generator</button>
-      <div id="fact-container" class="center"></div>
-      <div id="menu">
-        <a href="education.html" class="menu-option">
-          <div id="education" class="center">
-            <p class="menu-label">Education</p>
-            <img src="/images/books.jpg" alt="Books" class="menu-image">
+
+<head>
+	<meta charset="UTF-8">
```

Comment:
It looks like it converted everything to tabs, was this what the autoformat in the editor did?